config/config.go

```diff
@@ -121,6 +129,11 @@ var defaultConf = Config{
 		XHost: "0.0.0.0",
 		XPort: 14000,
 	},
+	PlanCache: PlanCache{
+		Enabled:  true,
```

Comment:
Set default to true is not safe for now.

Reply:
only for the ci test, will be disabled in the last commit of this pr